onnxmltools/convert/sklearn/operator_converters/KNN.py

```diff
@@ -50,45 +59,55 @@ def convert_sklearn_knn(scope, operator, container):
     #               V
     #   ARRAYFEATUREEXTRACTOR <- training_labels[M]
     #           |
-    #           V              (KNN Regressor)
-    #          topk_labels[K] ------------------> REDUCEMEAN --> output[1] 
-    #                    | 
-    #                   /|\
-    #                  / | \(KNN Classifier)
-    #                 /  |  \
-    #                /   |   \
-    #               /    |    \__
-    #               |    |       |
-    #               V    V       V
-    # label0 -> EQUAL  EQUAL ... EQUAL <- label(C-1)
-    #            |       |          |
-    #            V       V          V
-    # output_label_0[C] ...       output_label_(C-1)[C]
-    #            |       |          |
-    #            V       V          V
-    #          CAST    CAST    ... CAST 
-    #            |       |          |
-    #            V       V          V
-    # output_cast_label_0[C] ...  output_cast_label_(C-1)[C]
-    #            |       |          |
-    #            V       V          V
-    #      REDUCESUM  REDUCESUM ... REDUCESUM
-    #            |       |          |
-    #            V       V          V
-    # output_label_reduced_0[1] ... output_label_reduced_(C-1)[1]
-    #           \        |           /
-    #            \____   |      ____/ 
-    #                 \  |  ___/ 
-    #                  \ | / 
-    #                   \|/
-    #                    V
-    #                 CONCAT --> concat_labels[C]
-    #                               |
-    #                               V
-    #                           ARGMAX --> predicted_label[1] 
-    #                                       |
-    #                                       V
-    #            output[1] <--- ARRAYFEATUREEXTRACTOR <- classes[C]
+    #           V                        (KNN Regressor)
+    #          topk_labels[K] ----------------------------> REDUCEMEAN --> output[1] 
+    #                    |        |___________________________________________ 
```

Comment:
Nit. This line looks a bit wired. I would suggest
```suggestion
    #                    |        |
    #                             `-------------------------------------------.
```